XamlControlsGallery/ControlPages/AutoSuggestBoxPage.xaml

```diff
@@ -19,6 +19,47 @@
                 Width="300" AutomationProperties.Name="Basic AutoSuggestBox"/&gt;
                 </x:String>
             </local:ControlExample.Xaml>
+            <local:ControlExample.CSharp>
+                <x:String xml:space="preserve">
+// List of cats
+private List&lt;string&gt; Cats = new List&lt;string&gt;()
+{
+    "Abyssinian",
+    "Aegean",
+    "American Bobtail",
+    ...
+};
+
+// Handle text change and present suitable items
+private void AutoSuggestBox_TextChanged(AutoSuggestBox sender, AutoSuggestBoxTextChangedEventArgs args)
+{
+    var suitableItems = new List&lt;string&gt;();
+    var splitText = sender.Text.ToLower().Split(" ");
+    foreach(var cat in Cats)
+    {
+        var found = splitText.All((key)=>
+        {
+            return cat.ToLower().Contains(key);
+        });
+        if(found)
+        {
+            suitableItems.Add(cat);
+        }
+    }
+    if(suitableItems.Count == 0)
+    {
+        suitableItems.Add("No items found");
```

Comment:
Please update the text here as well.

Reply:
Always forget to update that too, fixed now.